### DISCUSSION TITLE
Will you add Towny as a direct download on Spigot?

### DISCUSSION THREAD

**Discussion by Peace1510** [TownyAdvanced/Towny#5240] [Q&A] [answered]
sir am the broken smile guy see ask him whoever gives you your income sir don't ask me to ask aternos to raise income pls do something for this problem i love this plugin and i can't remake my server pls i am requesting  you add it pls

**Comment by LlmDl** [comment, marked as answer]
I'm not really planning on adding Towny as a direct download on spigot. We want people on the github, seeing the wiki, seeing the issue tickets, reading the install guides, reading the patch notes.

**Reply by Peace1510** [reply]
but u can do the both direct on spigot and github so everyone can join and it will increase your download rate also so both you and aternos user will be happy huh

**Reply by Peace1510** [reply]
and the thing is almost everyone want this plugin on the server they love this plugin a lot so you should make it direct download and external download . i will make sure that if anyone use ur plugin from my side i will ask him to reach out to github ok.

**Reply by Peace1510** [reply]
and reply as soon as possible or to take any action further . i hope you get it and solve the problem .

**Reply by Peace1510** [reply]
[action] like if i can upload it on spigot from my user i will download the whole thing and upload it by my user if u can give me perms to do that i will make it direct download see try making it direct no one knows aternos raises your income for that no one knows. besides this is a very popular plugin it will gain more user for u and aternos it is good for all.

**Reply by Peace1510** [reply]
or try talking with aternos your a developer they can change their mind add github to the java list see try it i am waiting for your reply .reply as soon as possible .

**Reply by LlmDl** [reply]
I guess I'll fall back to the joke answer since you ignored my answer above: if its a big deal I'm guessing it's something worth paying for. So when I start to get a sick kickback or I get to 100 Sponsors I'll consider adding Towny as a direct download to Spigot. Quid pro quo.
